src/main/java/ch/njol/skript/effects/EffWardenDisturbance.java

```diff
@@ -0,0 +1,61 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Direction;
+import ch.njol.util.Kleenean;
+import org.bukkit.Location;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Warden;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Make Disturbance")
+@Description("Make a warden sense a disturbance at a location, causing the warden to investigate that area.")
```

Comment:
Does it add any anger? Does it override any existing disturbances? Will the warden drop whatever it's doing to investigate? These kinds of details were what I was wanting when i asked you to add more detail, not just reword it.